posthog/settings/dynamic_settings.py

```diff
@@ -112,3 +114,7 @@
     "EMAIL_REPLY_TO",
     "ASYNC_MIGRATIONS_OPT_OUT_EMAILS",
 )
+
+SECRET_SETTINGS = [
```

Comment:
Future generations will super appreciate a comment what does secrecy mean in this case :D (i.e. are these encrypted, are the values hidden, can the user edit these at all)

Reply:
Absolutely, great point!